tests/tracing/test_trace_manager.py

```diff
@@ -2,8 +2,10 @@
 from threading import Thread
 from unittest import mock
 
+from opentelemetry import trace as trace_api
```

Comment:
```suggestion
from opentelemetry import trace as otel_trace
```

Is this more clear?

Reply:
not a blocker

Reply:
It's weird convention that many other libraries do somehow.... I agree it's more clear, but probably too many changes are required for this PR. Let's do as a follow-up.